secret/secrets_manager_vault.go

```diff
@@ -12,18 +13,18 @@ import (
 // BasicSecretsManager provides a Vault implementation backed by Amazon Secrets
```

Comment:
`cocoa.Vault`

Reply:
Also, will it always be an amazon secrets manager?

Reply:
Yes, this implementation is backed by a Secrets Manager client.